### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by initiater** [open-mmlab/mmdetection#9985]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmdetection

### Environment

/home/scholar/mmcv/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
sys.platform: linux
Python: 3.8.16 (default, Mar  2 2023, 03:21:46) [GCC 11.2.0]
CUDA available: True
GPU 0: NVIDIA GeForce RTX 3090 Ti
CUDA_HOME: /usr/local/cuda-11.3
NVCC: Cuda compilation tools, release 11.3, V11.3.58
GCC: gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
PyTorch: 1.12.0
PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201402
  - Intel(R) oneAPI Math Kernel Library Version 2021.4-Product Build 20210904 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.6.0 (Git Hash 52b5f107dd9cf10910aaa19cb47f3abf9b349815)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.3
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=compute_37
  - CuDNN 8.3.2  (built against CUDA 11.5)
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.3, CUDNN_VERSION=8.3.2, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.12.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.13.0
OpenCV: 4.7.0
MMCV: 1.7.1
MMCV Compiler: GCC 9.4
MMCV CUDA Compiler: 11.3
MMDetection: 2.28.2+e9cae2d

this is result that i run python mmdet/utils/collect_env.py

### Reproduces the problem - code sample

2023-03-18 14:01:05,618 - mmdet - INFO - Epoch [4][1850/4624]	lr: 1.000e-03, eta: 8:35:48, time: 0.324, data_time: 0.007, memory: 8330, loss_rpn_cls: 0.0024, loss_rpn_bbox: 0.0057, loss_cls: 0.0530, acc: 98.1929, loss_bbox: 0.0608, loss: 0.1219
2023-03-18 14:01:21,727 - mmdet - INFO - Epoch [4][1900/4624]	lr: 1.000e-03, eta: 8:35:31, time: 0.322, data_time: 0.007, memory: 8330, loss_rpn_cls: 0.0017, loss_rpn_bbox: 0.0051, loss_cls: 0.0515, acc: 98.2642, loss_bbox: 0.0578, loss: 0.1161
Segmentation fault (core dumped)

While running, and then suddenly this problem occurs

### Reproduces the problem - command or script

i use python train.py my_maskrcnn.py

my_maskrcnn.py inherits from the  ../configs/mask_rcnn

### Reproduces the problem - error message

I tried using gdb to locate the error, but when it comes to source code, I can't resolve it

### Additional information

This problem has occurred many times. I have reinstalled cuda, cudnn, and pythoch, but it still cannot be resolved.

However, there is a problem that an error will be reported in my dataset, Corrupt JPEG data: 1022 extended bytes before marker 0xd9. I looked up and found that this does not matter, but I have some doubts.

**Comment by initiater** [comment]
Corrupt JPEG data:44 extraneous bytes before marker 0xd9,once this error occurs, either a segment error occurs or the screen is stuck

**Comment by freshn** [comment]
I had a similar problem:
'Corrupt JPEG data: 514 extraneous bytes before marker 0xd9 
Corrupt JPEG data: premature end of data segment'
when running the mmdetection codes. Have you solved this? It didn't stop the training but was super annoying because it made the log file a mess.

**Comment by lhlever** [comment]
You can try different cuda versions.
My original environment was python3.10+cuda11.8+pytorch2.10.
Segmentation fault (core dumped) occurred during training.
If cuda is changed to 11.7, training can be done normally without changing other versions.
